__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require("../app.js")
+const db = require("../db/connection.js")
+const request = require("supertest")
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data")
+
+
+beforeEach(() => {
+    return seed(data)
+ })
+ 
+afterAll(() => {
+    return db.end()
+ })
+
+ describe("/api/topics", () => {
+    test("GET 200: responds with an array of topic objects, each with slug and description properties", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.length).toBe(3)
+            body.forEach((topic) => {
+                expect(topic).toHaveProperty("description")
```

Comment:
a better jest matcher for testing objects, especially as you'll start testing bigger ones in the next few tickets, is .toMatchObject or .objectContaining. I'd suggest having a go at using them on future tickets

Reply:
thanks for the comments Roisin, I did try toMatchObject but it wasn't working the way I expected it to so resorted to using toHaveProperty but I can try it again